bitnami/kiam/README.md

```diff
@@ -0,0 +1,375 @@
+| `server.nodeSelector`                       | Node labels for pod assignment                                                              | `{}` (evaluated as a template)           |
+| `server.tolerations`                        | Tolerations for pod assignment                                                              | `[]` (evaluated as a template)           |
+| `server.podLabels`                          | Extra labels for kiam pods                                                                  | `{}`                                     |
+| `server.podAnnotations`                     | Annotations for kiam pods                                                                   | `{}`                                     |
+| `server.priorityClassName`                  | Server priorityClassName                                                                    | `nil`                                    |
+| `server.lifecycleHooks`                     | LifecycleHooks to set additional configuration at startup.                                  | `{}` (evaluated as a template)           |
+| `server.extraVolumeMounts`                  | Optionally specify extra list of additional volumeMounts for kiam container(s)              | `[]`                                     |
+| `server.extraVolumes`                       | Optionally specify extra list of additional volumes for kiam pods                           | `[]`                                     |
+| `server.initContainers`                     | Add additional init containers to the kiam pods                                             | `{}` (evaluated as a template)           |
+| `server.sidecars`                           | Add additional sidecar containers to the kiam pods                                          | `{}` (evaluated as a template)           |
+
+### kiam agent parameters
+
+| Parameter                                   | Description                                                                                | Default                                  |
+|---------------------------------------------|--------------------------------------------------------------------------------------------|------------------------------------------|
+| `agent.enabled`                             | Deploy the kiam agent                                                                      | `true`                                   |
+| `agent.containerPort`                       | HTTPS port to expose at container level                                                    | `8443`                                   |
+| `agent.allowRouteRegexp`                    | Regexp with the allowed paths for agents to redirect                                       | `nil`                                    |
```

Comment:
I would suggest `allowedRoutesRegExp` since it looks more appropriate for what it supposedly does

Reply:
The KIAM flag is like this
```
--allow-route-regexp
```

For consistency with the application, I'd leave the same name, but I will use RegExp instead of Regexp